cupy/core/core.pyx

```diff
@@ -3571,28 +3571,34 @@ cpdef ndarray matmul(ndarray a, ndarray b, ndarray out=None):
     for i in la:
         batchCount *= i
 
-    ap = _mat_ptrs(a)
-    bp = _mat_ptrs(b)
-    outp = _mat_ptrs(out_view)
+    # ap = _mat_ptrs(a)
+    # bp = _mat_ptrs(b)
+    # outp = _mat_ptrs(out_view)
+
+    strideA = int(a.shape[-2] * a.shape[-1])
```

Comment:
How about using `_strides` attribute?

Reply:
Thank you for your comment.
I tried to use `_strides` attribute, but the unit was `byte`, so that `shape` is used here.

Reply:
All right, fixed as you suggested!